File: dhandho-jsclient/src/app/service/logger.service.ts
import { Injectable } from '@angular/core';

@Injectable()
export class LoggerService {

    public info( msg: any ) {
        console.info( msg );
    }

    public trace( msg?: any, ...args: any[] ) {
        console.debug( msg, args );
    }

    public debug( msg: any, ...args: any[] ) {
        console.debug( msg, args );
    }

    public log( msg: any ) {
        console.log( msg );
    }

    public warn( msg: any ) {
        console.warn( msg );
    }

    public error( msg: any ) {
        console.error( msg );
    }
}

File: dhandho-jsclient/src/main/java/app/dhojsw/ng/service/LoggerService.java
package app.dhojsw.ng.service;

import static def.dom.Globals.console;

import def.angular.core.Injectable;

@Injectable()
public class LoggerService {

	public void info(Object msg, Object... args) {
		console.info(msg, args);
	}

	public void debug(Object msg, Object... args) {
		console.debug(String.valueOf(msg), args);
	}

	public void trace(Object msg, Object... args) {
		console.info(msg, args);
	}

	public void warn(Object msg, Object... args) {
		console.warn(msg, args);
	}

	public void error(Object msg, Object... args) {
		console.info(msg, args);
	}

}
